### ISSUE TITLE
Round `aria-valuetext` for all sliders to match the precision used in the view.

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#1119]
> - [ ] Round `aria-valuetext` for all sliders to match the precision used in the view.

 _Originally posted by @arouinfar in [#1103](https://github.com/phetsims/circuit-construction-kit-common/issues/1103#issuecomment-3643123642)_

**Comment by arouinfar** [comment]
> How shall we address sliders that do not have a value shown in the view?

For wire resistivity, let's round to 4 decimal places. 

The maximum resistivity value is 0.0168, and the minimum 1e-10, but one shift step increases the value to 0.00016... so 4 decimal places seems good enough, though not sure about what will happen at the minimum value.

Are there any other qualitative sliders we need to address @samreid?

**Comment by arouinfar** [comment]
@samreid when using the a11y view I noticed that the `aria-valuetext` for the sliders in the Advanced AccordionBox are missing units:
- [ ] Wire resistivity: ohm-meters 
- [ ] Battery/Source resistance: ohms

**Comment by arouinfar** [comment]
I reviewed all of the sliders in the ac and dc flavors, and all the `aria-valuetext` looks good except for the Battery/Source Resistance at the min value. The `aria-valuetext` says the value is zero, but really it's 0.0001. For Wire Resistivity, the minimum value uses scientific notation and is showing up just fine. Can you do the same for Battery/Source Resistance?
